tripleblindmarket/private-kit
# Multilingual support

Added setup for supporting multiple languages.
For now added content of Hindi alongside English for demonstration

**Created folders for -** 
1. English 
2. Hindi
3. German
4. Italian
5. Portuguese
6. French

**Test -** 

1. Change your device language to Hindi (as only hindi content is now added, rest other languages have same content as English)

2. Check the app for all the texts

**Contribute -** 

Each language has a folder, with json files.
Replace english content from these json files with the selected languages
